websocket/pinger.go

```diff
@@ -4,33 +4,27 @@ import (
 	"fmt"
 	"raccoon/logger"
 	"raccoon/metrics"
+	"raccoon/websocket/connection"
 	"time"
-
-	"github.com/gorilla/websocket"
 )
 
-type connection struct {
-	uniqConnID string
-	conn       *websocket.Conn
-}
-
-//Pinger is a worker groroutine that pings the connected peers based on ping interval.
-func Pinger(c chan connection, size int, PingInterval time.Duration, WriteWaitInterval time.Duration) {
+//Pinger is worker that pings the connected peers based on ping interval.
+func Pinger(c chan connection.Conn, size int, PingInterval time.Duration, WriteWaitInterval time.Duration) {
 	for i := 0; i < size; i++ {
 		go func() {
-			cSet := make(map[string]*websocket.Conn)
+			cSet := make(map[connection.Identifer]connection.Conn)
 			timer := time.NewTicker(PingInterval)
```

Comment:
can we please change this to ticker because the `Timer` is also a separate entity in `time` package.

Reply:
I don't understand this. Is there another Ticker?

Reply:
So there is https://pkg.go.dev/time#Timer to in Go, this one here is a ticker